lib/vsc/utils/timestamp_pid_lockfile.py

```diff
@@ -93,12 +94,12 @@ def acquire(self):
                     self.logger.warning('Obsolete lockfile detected at %s: pid = %d, timestamp = %s' % (self.path, pid, time.ctime(timestamp)))
                     try:
                         _write_pid_timestamp_file(self.path)
-                    except OSError, err:
-                        self.logger.error('Unable to obtain lock on %s' % (self.path))
-                        raise LockFailed
-                else:
-                    self.logger.error('Unable to obtain lock on %s' % (self.path))
-                    raise LockFailed
+                        doraise = False
+                    except:
+                        pass
+            if doraise:
+                self.logger.error('Unable to obtain lock on %s: %s' % (self.path, err))
+                raise LockFailed
 
```

Comment:
so this changes the behaviour? in that case, add a test, so it won't be reintroduced in the future